generate-release-notes.sh

```diff
@@ -2,56 +2,112 @@
 
 set -eo pipefail
 
+# Flags
+AL2_GPU_NVIDIA_VERSION=""
+AL2_GPU_CUDA_VERSION=""
+AL1_CONTAINERD_VERSION=""
+EXCLUDE_AMI=""
+
 usage() {
-    echo "Usage:"
-    echo "  $0 AL2_GPU_NVIDIA_VERSION AL2_GPU_CUDA_VERSION [AL1_CONTAINERD_VERSION]"
-    echo "  AL1_CONTAINERD_VERSION should not be provided if there is no AL1 AMI release"
-    echo "Example:"
-    echo "  $0 470.182.03 11.4 1.4.13 # AL1 release included"
-    echo "  $0 470.182.03 11.4        # No AL1 release"
+    cat <<-EOF
+Usage:
+  $0
+
+Options:
+	--al2-gpu-nvidia-ver  (Optional) AL2 GPU NVIDIA version. If specified, then --al2-gpu-cuda-ver option is also required to be specified.
+	--al2-gpu-cuda-ver    (Optional) AL2 GPU CUDA version. If specified, then  --al2-gpu-nvidia optin is also required to be specified.
+	--al1-containerd-ver  (Optional) AL1 containerd version.
```

Comment:
nit: should we retain the note`AL1_CONTAINERD_VERSION should not be provided if there is no AL1 AMI release`?

Reply:
Thanks for reviewing and for your comment!

I personally think since we have explicitly marked AL 1 containerd version as "(Optional)" here that it's redundant/overly verbose to keep that note.

However, I don't have a very strong preference either way. If you feel strongly about this, I can follow up later add this back in, as well as explicit notes that AL2 GPU NVIDIA version and AL2 GPU CUDA versions should not be provided if there is no AL2 GPU release.

Reply:
sgtm!